helm/README.md

```diff
@@ -0,0 +1,284 @@
+# ModelExpress Helm Chart
+
+This Helm chart deploys ModelExpress, a model serving and management platform, to Kubernetes.
+
+## Prerequisites
+
+- Kubernetes 1.19+
+- Helm 3.0+
+- Access to NVIDIA Container Registry (nvcr.io) for pulling the ModelExpress image
+
+## Installation
+
+### 1. Create NVIDIA Container Registry Secret
+
+Before installing the chart, you must create a Kubernetes secret to access the private NVIDIA Container Registry (nvcr.io). The default image requires authentication.
+
+```bash
+# Create the secret in your target namespace
+kubectl create secret docker-registry nvcr-secret \
+  --docker-server=nvcr.io \
+  --docker-username='$oauthtoken' \
+  --docker-password='YOUR_NVCR_API_KEY' \
+  --docker-email='your-email@nvidia.com' \
+  --namespace=your-namespace
+
+# Or create it in the default namespace if you plan to use that
+kubectl create secret docker-registry nvcr-secret \
+  --docker-server=nvcr.io \
+  --docker-username='$oauthtoken' \
+  --docker-password='YOUR_NVCR_API_KEY' \
+  --docker-email='your-email@nvidia.com'
+```
+
+**Important Notes:**
+- Replace `YOUR_NVCR_API_KEY` with your actual NVIDIA Container Registry API key
+- The username must be `$oauthtoken` (literal string, not a variable)
+- The email should be your NVIDIA email address
+- The secret name `nvcr-secret` is referenced in the default values
+
+### 2. Update Values to Use the Secret
+
+The default `values.yaml` already includes the image pull secret configuration:
+
+```yaml
+# values.yaml (default)
+imagePullSecrets:
+  - name: nvcr-secret
+```
+
+If you're using a custom values file, ensure it includes this configuration or the deployment will fail with image pull errors.
+
+### 3. Add the Helm repository (if using a repository)
+
+```bash
+helm repo add model-express https://your-repo-url
+helm repo update
+```
+
+### 4. Install the chart
+
+```bash
+# Install with default values
+helm install my-model-express ./helm
+
+# Install with custom values
+helm install my-model-express ./helm -f values.yaml
+
+# Install in a specific namespace
+helm install my-model-express ./helm --namespace model-express --create-namespace
+```
+
+## Configuration
+
+The following table lists the configurable parameters of the ModelExpress chart and their default values.
+
+| Parameter | Description | Default |
```

Comment:
Please make tables readable in plain-text as well; after all that's entire point of markdown.